PxWeb/Code/Api2/DataSource/PxFile/PxFileDataSource.cs

```diff
@@ -83,6 +87,27 @@ public PxFileDataSource(IPxFileConfigurationService pxFileConfigurationService,
             return menu.CurrentItem;
         }
 
+        public Codelist? GetCodelist(string id, string language)
+        {
+            Codelist? codelist = null;
+            
+            if (string.IsNullOrEmpty(id))
+            {
+                return codelist;
+            }
+
+            id = id.Replace("agg_", "");
```

Comment:
We sould olny remove agg_ from the beginning of the id.

Reply:
Now uses substring instead